logs/artifacts/000__nfd_operator__presubmit_job/_ansible.cmd

```diff
@@ -0,0 +1 @@
+ansible-playbook -vv -e nfd_operator_git_repo=https://github.com/openshift/cluster-nfd-operator.git -e presubmit_pull_number=184 -e presubmit_pull_base_ref=master -e presubmit_pull_pull_sha=4362a79319a3c757905b1d9e72dc2f2110abd256 -e nfd_operator_image_tag=ci-image -e artifact_dir=/home/eduardo/sdk/github/openshift-psap/ci-artifacts/logs/artifacts -e artifact_extra_logs_dir=/home/eduardo/sdk/github/openshift-psap/ci-artifacts/logs/artifacts/000__nfd_operator__presubmit_job playbooks/nfd_operator_presubmit.yml
```

Comment:
Looks like you accidentally uploaded your logs

Reply:
ups

Reply:
looking better now

Reply:
Logs still here it seems

Reply:
👍🏻 logs are gone